bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties

```diff
@@ -94,8 +94,8 @@ thing-type.config.max.thermostat.ecoTemp.label = Absenktemperatur
 thing-type.config.max.thermostat.ecoTemp.description = Gibt die Absenktemperatur (in °C) des Heizkörperthermostats an.
 thing-type.config.max.thermostat.group.actions.label = Aktionen
 thing-type.config.max.thermostat.group.actions.description = Einstellungen für Aktionen.
-thing-type.config.max.thermostat.group.binding.label = Binding
-thing-type.config.max.thermostat.group.binding.description = Einstellungen für das Binding.
+thing-type.config.max.thermostat.group.addon.label = Binding
+thing-type.config.max.thermostat.group.addon.description = Einstellungen für das binding.
```

Comment:
```suggestion
thing-type.config.max.thermostat.group.addon.description = Einstellungen für das Binding.
```

Reply:
@J-N-K - I went through all files now. Sorry for the messy review. I believe this is my last finding.